daisy_workflows/image_build/debian/build_fai.py

```diff
@@ -49,6 +50,53 @@ def mycopytree(src, dst):
     return dst
 
 
+# get number of bytes to offset when mounting the raw disk image file
```

Comment:
we should not be modifying the build steps for debian; sbom generation should be its own separate process